tests/test_load_audio.py

```diff
@@ -18,41 +18,46 @@ def test_load_wav(benchmark):
 def test_load_au(benchmark):
     # read au file
     path = Path(signalworks.__file__).parents[1] / "data" / "test.au"
-    wave = benchmark(Track.read, path)
-    assert np.any(wave.value > 0)
-    assert wave.fs == 44100
+    with pytest.raises(Exception):
```

Comment:
Current configuration will ensure this test _always_ passes.  Instead we want to skip this test if `soundfile` is not installed, if `soundfile` is installed, we want to run this test normally.  To do that, we use `pytest.importorskip`.
2
`soundfile = pytest.importorskip("soundfile", reason="If soundfile is not installed, this test will fail")`